### PR TITLE
Successfully added border-radius to button and images, changes font and background color of heading, adding some paddings and margin and added onhover effect also.

### PR DESCRIPTION
**Pull request by ankur0904**
I made some changes in the frontend files of this project. This is a webpage that has the potential to manage to vote.
One of them is I added the border-radius to both the images due to the images exactly fit with the background image.
And next things are about the buttons I made some changes in the CSS part of the button, e.g. I added padding to the button to increase its visibility etc.
I managed all the positioning of all the div so that all div exactly fit with their place.

This is all about my contribution.

### CODE REVIEW

File: Level0/Voting/README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Voting applcation
 
-
```
Comment (Vikash-8090-Yadav) [inline Level0/Voting/README.md:3]: add the image of the   voting site aftet the Heading voting application

**Comment by Vikash-8090-Yadav** [comment]
Work on the review ..Now all set to merge

**Comment by ankur0904** [comment]
Screenshot from 2022-10-02 13-51-49

**Comment by ankur0904** [comment]
I updated the README.md

**Comment by Vikash-8090-Yadav** [comment]
Great @ankur0904 . Looking forward for your more contribution from ur side . 
Hapoy hacktoberfest

**Pull request merged**